products/statement-generator/src/components/Input.tsx

```diff
@@ -101,16 +102,39 @@ const InputArea: React.FC<InputFieldProps> = ({
   adornment,
   className = '',
   shortWidth = false,
+  errorText,
 }) => {
   const utilityClasses = useUtilityStyles();
   const classes = useStyles({
     disabled,
     useShort: shortWidth || type === 'number',
   });
 
-  const [valid, isValid] = useState(false);
-  const checkValid = (e: string) => {
-    isValid(e.length > 0);
+  const [valid, isValid] = useState(true);
```

Comment:
can you rename the setter here? It's a little confusing to have isValid when you already have a valid variable.